src/logic/equiv/set.lean

```diff
@@ -362,6 +362,14 @@ protected def prod {α β} (s : set α) (t : set β) :
   ↥(s ×ˢ t) ≃ s × t :=
 @subtype_prod_equiv_prod α β s t
 
+/-- The set `set.pi set.univ s` is equivalent to `Π a, s a`. -/
+@[simps] protected def pi {α : Type*} {β : α → Type*} (s : Π a, set (β a)) :
```

Comment:
```suggestion
@[simps] protected def pi_univ {α : Type*} {β : α → Type*} (s : Π a, set (β a)) :
```
since I think we have a handful of lemmas that use the same terminology.

Reply:
(also, we might want `pi` for `pi t s ≃ Π a \in t, s a`

Reply:
This is called `univ_pi` elsewhere. We can't have `pi t s ≃ Π a \in t, s a` because `pi t s` is the set of functions such that `f i ∈ s i` whenever `i ∈ t`; in particular, if `t` is the empty set, then `pi t s = univ`.